stackit/internal/services/objectstorage/bucket/resource.go

```diff
@@ -155,8 +155,15 @@ func (r *bucketResource) Create(ctx context.Context, req resource.CreateRequest,
 	ctx = tflog.SetField(ctx, "project_id", projectId)
 	ctx = tflog.SetField(ctx, "bucket_name", bucketName)
 
-	// Create new recordset
-	_, err := r.client.CreateBucket(ctx, projectId, bucketName).Execute()
+	// handle project init
+	err := r.enableProject(ctx, &model)
+	if resp.Diagnostics.HasError() {
+		core.LogAndAddError(ctx, &resp.Diagnostics, "Error creating credentials group", fmt.Sprintf("Enabling object storage project before creation: %v", err))
```

Comment:
"Error creating credentials group" --> "Error creating bucket"

Reply:
Oh the good old copy paste strikes again